curvefs/src/metaserver/inode_storage.cpp

```diff
@@ -60,9 +60,9 @@ MetaStatusCode InodeStorage::Insert(const Inode& inode) {
     WriteLockGuard writeLockGuard(rwLock_);
     Key4Inode key(inode.fsid(), inode.inodeid());
     std::string skey = conv_.SerializeToString(key);
-    if (FindKey(skey)) {
-        return MetaStatusCode::INODE_EXIST;
-    }
+    // if (FindKey(skey)) {
```

Comment:
it depends on https://github.com/opencurve/curve/pull/1561, and the `FindKey` is used to check whether the key exists, and I think the logic also has been removed by https://github.com/opencurve/curve/pull/1561, because the key sets will be very large.